src/main/scala/beam/sim/BeamSim.scala

```diff
@@ -126,15 +120,19 @@ class BeamSim @Inject()(private val actorSystem: ActorSystem,
   }
 
   private def cleanupVehicle() = {
-    for ( (_, actorRef) <- vehicleActors) {
-      logger.debug(s"Stopping ${actorRef.path.name} ")
+    logger.info(s"Stopping  BeamVehicle actors")
```

Comment:
Agreed, there is a lot of logging everywhere that falls into the same category. But I'd rather clean up logging after we've gotten our near-term features implemented since they aid in debugging...